src/node.h

```diff
@@ -291,9 +291,38 @@ class NODE_EXTERN MultiIsolatePlatform : public v8::Platform {
 NODE_EXTERN void SetIsolateCreateParams(v8::Isolate::CreateParams* params,
                                         ArrayBufferAllocator* allocator
                                             = nullptr);
+
+enum IsolateSettingsFlags {
+  MESSAGE_LISTENER_WITH_ERROR_LEVEL = 1 << 0,
+  DETAILED_SOURCE_POSITIONS_FOR_PROFILING = 1 << 1
+};
+
+struct IsolateSettings {
```

Comment:
can you put in a warning that overriding these callbacks may interfere with node core functionality?